Swift Code Challenge/Views/UserDetailsView.swift

```diff
@@ -0,0 +1,119 @@
+                .ignoresSafeArea(edges: .top)
+            VStack{
+                CircleImage(userImage: user.picture.large ?? "")
+                
+                contentBody(user: user)
+            }
+            .offset(y: -200)
+            .padding(.bottom, -130)
+            
+            Spacer()
+            
+            Text("made by @luizmello.dev - click me!")
+                .onTapGesture {
+                    openInstagram(instagramHandle: "luizmello.dev")
+                }
+                .padding(.vertical)
+        }
+    }
+    func openInstagram(instagramHandle: String) {
+        guard let url = URL(string: "https://instagram.com/\(instagramHandle)")  else { return }
+        if UIApplication.shared.canOpenURL(url) {
+            if #available(iOS 10.0, *) {
+                UIApplication.shared.open(url, options: [:], completionHandler: nil)
+            } else {
+                UIApplication.shared.openURL(url)
+            }
+        }
+    }
+}
+
+struct contentBody: View {
+    @State var result: Result<MFMailComposeResult, Error>? = nil
+    @State var isShowingMailView = false
+    var user: Person
+    
+    var body: some View {
+        VStack(spacing: 5) {
+            Text("\(user.name.first ?? "Luiz") \(user.name.last ?? "Mello")")
+                .bold()
+                .font(.title)
+            HStack() {
+                Text("\(user.location.street?.name ?? "SADSAD"),")
+                    .font(.body)
+                    .foregroundColor(.secondary)
+                Text("\(user.location.city ?? "asdsad") - \(user.location.country ?? "asdad")")
+                    .font(.body)
+                    .bold()
+                    .foregroundColor(.secondary)
+            }
+            .padding(.bottom, 70)
+            
+            Button(action: {
+                let tel = "mailto:"
+                var formattedString = tel + user.email
+                let url: NSURL = URL(string: formattedString)! as NSURL
+                
+                UIApplication.shared.open(url as URL)}) {
+                    HStack {
+                        Image(systemName: "mail")
+                            .font(.body)
+                        Text(user.email)
+                            .fontWeight(.semibold)
+                            .font(.body)
+                    }
+                    .padding()
+                    .foregroundColor(.white)
+                    .background(Color.blue)
+                    .cornerRadius(40)
+                }
+                .padding()
+            
+            Button(action: {
+                
+                // validation of phone number not included
+                let dash = CharacterSet(charactersIn: "-")
+                
+                let cleanString =
+                user.phone.trimmingCharacters(in: dash)
+                
+                let tel = "tel://"
+                var formattedString = tel + cleanString
```

Comment:
No need for this to be a `var`.

```suggestion
                let formattedString = tel + cleanString
```